plugins/http_plugin/include/eosio/http_plugin/beast_http_session.hpp

```diff
@@ -159,7 +159,7 @@ class beast_http_session : public detail::abstract_conn {
             error_results results{static_cast<uint16_t>(http::status::not_found), "Not Found",
                                   error_results::error_info( fc::exception( FC_LOG_MESSAGE( error, "Unknown Endpoint" ) ),
                                                              http_plugin::verbose_errors() )};
-            send_response( fc::json::to_string( results, fc::time_point::now() + plugin_state_->max_response_time ),
+            send_response( fc::json::to_string( results, fc::time_point::maximum() ),
```

Comment:
Why this change?

Reply:
Because the input to `to_string` is small and static so no possibility of json bomb.

Reply:
OK, thanks!